src/query_crypto_get_account_records.rs

```diff
@@ -0,0 +1,89 @@
+use super::{errors::PyException, transaction_receipt::PyTransactionReceipt, timestamp::py_date_time};
+use hedera::{
+    query::{Query, QueryCryptoGetAccountRecords},
+    TransactionRecord,
+    TransactionRecordBody,
+    Client, AccountId
+};
+use pyo3::prelude::*;
+use pyo3::types::PyDateTime;
+
+#[pyclass(name = QueryCryptoGetAccountRecords)]
+pub struct PyQueryCryptoGetAccountRecords {
+    inner: Query<Vec<TransactionRecord>>
```

Comment:
This is now written as `Query<QueryType>` if you've updated the Rust instead of `Query<OutputType>`